specs/src/test/groovy/com/cloudbees/pdk/hen/tests/CreateFolderSpec.groovy

```diff
@@ -0,0 +1,55 @@
+package com.cloudbees.pdk.hen.tests
+
+import com.cloudbees.pdk.hen.procedures.CreateFolder
+import spock.lang.Shared
+import spock.lang.Stepwise
+import spock.lang.Unroll
+
+@Stepwise
+class CreateFolderSpec  extends PluginTestHelper {
+
+    @Shared
+    CreateFolder createFolder = pluginWithoutConfig.createFolder
+    @Shared
+    def bucketName = "cbspec"
+    @Shared
+    def folderName = "cbspec-folder"
+
+    def doSetupSpec() {
+        configName = configName.toString().replace("configurations","pluginConfigurations")
+        bucketName += generator( (('a'..'z')+('0'..'9')).join(), 9 )
+        createBucket(bucketName)
+    }
+    def doCleanupSpec() {
+        deleteBucket(bucketName)
+        deleteProject("specs-EC-S3")
+        deleteProject("spec-configs-project-EC-S3")
+    }
+
+    def 'Create folder'() {
+        when:
+        def r = createFolder.config(configName)
+                .bucketName(bucketName)
+                .folderName(folderName)
+                .run()
+        then:
+        assert r.successful
```

Comment:
Job is successful, but was it? No assert on folder creation

Reply:
assertion added in last commit